homeassistant/components/light/flux_led.py

```diff
@@ -34,6 +34,8 @@
     vol.Optional(CONF_NAME): cv.string,
     vol.Optional(ATTR_MODE, default='rgbw'):
         vol.All(cv.string, vol.In(['rgbw', 'rgb'])),
```

Comment:
If the changes to add LEDENET protocol to flux_led pypi package allow for controlling all five channels (RGBWW), we need to consider adding RGBWW as a mode in the near future. For now, let's just get LEDENET added in working condition and stable for all users.